src/main/java/org/camunda/community/benchmarks/StartPiExecutor.java

```diff
@@ -42,6 +41,12 @@ public class StartPiExecutor {
     public void init() throws IOException {
         String variablesJsonString = tryReadVariables(config.getPayloadPath().getInputStream());
         benchmarkPayload = zeebeClientConfiguration.getJsonMapper().fromJsonAsMap(variablesJsonString);
+        benchmarkPayload.forEach((key, value)->{
+            if(value instanceof String && !((String) value).isEmpty() && ((String) value).contains("${RANDOM_NUMBER}")) {
+                benchmarkPayload.replace(key,
+                    ((String) value).replace("${RANDOM_NUMBER}", UUID.randomUUID().toString()));
```

Comment:
If it's a UUID the placeholder should probably be called `RANDOM_UUID`

Reply:
Changed it.